dev/Pager/PrototypePager/Themes/PrototypePager.xaml

```diff
@@ -0,0 +1,181 @@
+                                           Visibility="{Binding ElementName=FirstPageButton, Path=Visibility}"/>
+                                <Button x:Name="PreviousPageButton"
+                                        IsEnabled="False"
+                                        Style="{StaticResource NavigationBackButtonNormalStyle}"
+                                        FontFamily="Segoe MDL2 Assets"
+                                        Content="&#xE76B;"/>
+                                <TextBlock x:Name="PreviousPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
+                                           Text="{TemplateBinding PreviousPageButtonText}"
+                                           Visibility="{Binding ElementName=PreviousPageButton, Path=Visibility}"/>
+                            </StackPanel>
+                            <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal" Grid.Row="0" Grid.Column="1">
+                                    <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{TemplateBinding PrefixText}" />
+                                    <!--NumberBox Display-->
+                                    <controls:NumberBox x:Name="NumberBoxDisplay"
+                                                    HorizontalAlignment="Center"
+                                                    VerticalAlignment="Center"
+                                                    Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=NumberOfPages}"
+                                                    Minimum="1"
+                                                    Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntToDouble}, Path=SelectedIndex, Mode=TwoWay}"
+                                                    SmallChange="1"
+                                                    LargeChange="5"
+                                                    IsEnabled="False"
+                                                    Visibility="Collapsed" />
+                                    <!--ComboBox Display-->
+                                    <ComboBox x:Name="ComboBoxDisplay"
+                                          HorizontalAlignment="Center" VerticalAlignment="Center"
+                                          SelectedIndex="{Binding RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource IntMinusOne}, Path=SelectedIndex, Mode=TwoWay}"
+                                          ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=PagerTemplateSettings.Pages}"
+                                          IsEnabled="False"
+                                          Visibility="Collapsed"/>
+                                    <TextBlock VerticalAlignment="Center" Margin="5,0,5,0" Text="of"/>
```

Comment:
>Text="of" [](start = 91, length = 9)

keep in mind this will need to be localized, not needed for this check in.

Reply:
I have swapped it out for a "/" instead. Hopefully better for now.